templates/partners/partial/_homepage-form.html

```diff
@@ -1,6 +1,6 @@
 <div class="p-modal contact-form" style="display: none;" id="homepage-modal">
   <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
-    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+    <header class="p-modal__header" style="display: block; border-bottom: 0; margin-bottom: 0; padding-bottom: 0;">
```

Comment:
We actually have vanilla classes for some of these. See [padding](https://vanillaframework.io/docs/utilities/padding-collapse) and [margin](https://vanillaframework.io/docs/utilities/margin-collapse) collapse
```suggestion
    <header class="p-modal__header u-no-margin--bottom u-no-padding--bottom" style="display: block; border-bottom: 0;">
```